packages/mdc-ripple/_mixins.scss

```diff
@@ -184,18 +184,22 @@
   }
 }
 
-@mixin mdc-states-base-color($color, $query: mdc-feature-all()) {
+@mixin mdc-states-base-color($color: null, $query: mdc-feature-all()) {
   $feat-color: mdc-feature-create-target($query, color);
 
   &::before,
   &::after {
     @include mdc-feature-targets($feat-color) {
-      @if alpha(mdc-theme-prop-value($color)) > 0 {
-        @include mdc-theme-prop(background-color, $color, $edgeOptOut: true);
+      @if($color != null) {
+        @if alpha(mdc-theme-prop-value($color)) > 0 {
+          @include mdc-theme-prop(background-color, $color, $edgeOptOut: true);
+        } @else {
+          // If a color with 0 alpha is specified, don't render the ripple pseudo-elements at all.
+          // This avoids unnecessary transitions and overflow.
+          content: none;
+        }
       } @else {
-        // If a color with 0 alpha is specified, don't render the ripple pseudo-elements at all.
-        // This avoids unnecessary transitions and overflow.
-        content: none;
+        background-color: currentColor;
```

Comment:
Seems like unsafe assumption to use inherited color for ripple where the inherited color is unknown.

Reply:
In the majority of our components the background-color is currentColor. We will provide the mixin to override this. Also this IFF the $color argument is not provided, which was not an option before. Meaning this is backwards compatible.